client/src/stories/Button.js

```diff
@@ -1,50 +1,32 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import './button.css';
-
+import React from "react";
```

Comment:
Importing React is optional with React v17, but it is consistent in your code, so that looks good 👍 
In case you want to delete it, you can add `"react/react-in-jsx-scope": 0` to your ESLint rules to avoid eslint errors.